sycl/include/CL/sycl/item.hpp

```diff
@@ -104,6 +104,8 @@ template <int dimensions = 1, bool with_offset = true> class item {
 
   bool operator!=(const item &rhs) const { return rhs.MImpl != MImpl; }
 
+  void set_allowed_range(const range<dimensions> rnwi) { MImpl.MExtent = rnwi; }
```

Comment:
this new function is public.  Should it be?

Reply:
Good catch.  This seems to introduce a lot of new public functions.  I don't think we should be doing that.

Reply:
The whole point of this is to be transparent to the programmer, right?

Reply:
Fixed.